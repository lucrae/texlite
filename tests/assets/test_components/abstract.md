:abstract: This is my abstract