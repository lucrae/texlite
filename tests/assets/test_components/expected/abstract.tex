% meta
\documentclass[10pt]{extarticle}

% packages
\usepackage[utf8]{inputenc}
\usepackage[margin=1.6in]{geometry}
\usepackage{hyperref}
\usepackage{amsmath}
\usepackage{amssymb}

% preamble commands
\linespread{1.0}
\usepackage{graphicx}
\graphicspath{{tests/assets/test_components/}}

% document details
\date{}

% ------------------------------------------------------------------------------
\begin{document}
% ------------------------------------------------------------------------------

\begin{abstract}
This is my abstract
\end{abstract}

% ------------------------------------------------------------------------------
\end{document}
% ------------------------------------------------------------------------------

